class/defaults.yml

```diff
@@ -21,3 +21,10 @@ parameters:
 
     secrets: {}
     cert_manager_certs: {}
+
+    monitoring:
+      enabled: true
+      instance: ''
+      enableServiceMonitors:
+        ingress-operator: false
```

Comment:
The operator only has some metrics on the ingress canary, which can also be monitored through the ingress controller. I don't see much value in enabling it by default